Admin/GalleryAdmin.php

```diff
@@ -124,8 +124,13 @@ protected function configureFormFields(FormMapper $formMapper)
                 ))
                 ->add('enabled', null, array('required' => false))
                 ->add('name')
+                ->ifTrue($format)
                 ->add('defaultFormat', 'choice', array('choices' => $formats))
-            ->end()
+                ->ifEnd();
+
+        $formMapper->end();
```

Comment:
To be consistent with other with and end's calls this should be like before, nested at the same level of 
`->with('Options')`